JASP-Engine/JASP/R/descriptives.R

```diff
@@ -975,6 +992,73 @@ Descriptives <- function(jaspResults, dataset, options) {
   return(thePlot)
 }
 
+.descriptivesIntervalPlot <- function(dataset, options, variable) {
+
+  # Define custom y axis function
+  base_breaks_y <- function(x) {
+    b <- pretty(x)
+    d <- data.frame(x=-Inf, xend=-Inf, y=min(b), yend=max(b))
+    list(ggplot2::geom_segment(data=d, ggplot2::aes(x=x, y=y, xend=xend, yend=yend), size = 0.75, inherit.aes=FALSE), ggplot2::scale_y_continuous(breaks=b))
+  }
+
+  thePlot <- createJaspPlot(title=variable, width=options$plotWidth, height=options$plotHeight)
+
+  errorMessage <- .descriptivesCheckPlotErrors(dataset, variable, obsAmount = "< 1")
+  if (!is.null(errorMessage)) {
+    thePlot$setError(gettextf("Plotting not possible: %s", errorMessage))
+  } else {
+    # we need to know which index in y is related to which index in the actual data, so we should not forget the NAs somehow, lets make a list of indices.
+    yWithNA         <- dataset[[.v(variable)]]
+    y               <- na.omit(dataset[[.v(variable)]])
+    yIndexToActual  <- y
+    yWithNAIndex    <- 1
+    yNoNAIndex      <- 1
+    while(yWithNAIndex <= length(yWithNA)) {
+
+      if(!is.na(yWithNA[[yWithNAIndex]])) {
+
+        yIndexToActual[[yNoNAIndex]] <- row.names(dataset)[[yWithNAIndex]]
+        yNoNAIndex                   <- yNoNAIndex + 1
+      }
+
+      yWithNAIndex <- yWithNAIndex + 1
+    }
+
+    if (is.null(dataset[[.v(options$splitby)]])){
+      group     <- factor(rep("",length(y)))
+      xlab      <- "Total"
+
+    } else {
+      group     <- as.factor(dataset[[.v(options$splitby)]])[!is.na(dataset[[.v(variable)]])]
+      xlab      <- options$splitby
+
+    }
+
+
+    plotDat <- data.frame(group = group, y = y)
+
+    cdata <- plyr::ddply(plotDat, "group", "summarise",
+                       mean = mean(y),
+                       lower = mean - qnorm(0.975) * (sd(y) / sqrt(length(y))),
+                       upper = mean + qnorm(0.975) * (sd(y) / sqrt(length(y))))
+
+     p <- ggplot2::ggplot(cdata, ggplot2::aes(x = group, y = mean, ymin = lower, ymax = upper)) +
+          ggplot2::stat_summary(fun.y = mean, geom = "point") +
+          ggplot2::geom_errorbar(width = .1)
+
+    ### Theming & Cleaning
+    yBreaks <- JASPgraphs::getPrettyAxisBreaks(y)
```

Comment:
```suggestion
    yBreaks <- JASPgraphs::getPrettyAxisBreaks(c(cdata[["lower"]], cdata[["upper"]]))
```
so the breaks here are based on the data `y` but not on the summary statistics of `y` (stored in `cdata`). As a consequence, the y-axis of the plots doesn't make sense: 

![image](https://user-images.githubusercontent.com/21319932/95554949-5a431680-0a11-11eb-99b4-268f664c3e71.png)